spec/yoti/share/attribute_issuance_details_spec.rb

```diff
@@ -62,4 +62,29 @@ def create_thirdparty_attribute_proto(token, expiry_date, *definitions)
     it 'sets a nil value for date' do
     end
   end
+
+  context 'with any RFC3339 date' do
+    [
+      ['2006-01-02T22:04:05Z', DateTime.new(2006, 1, 2, 22, 4, 5)],
+      ['2006-01-02T22:04:05.1Z', DateTime.new(2006, 1, 2, 22, 4, 5.1)],
+      ['2006-01-02T22:04:05.12Z', DateTime.new(2006, 1, 2, 22, 4, 5.12)],
+      ['2006-01-02T22:04:05.123Z', DateTime.new(2006, 1, 2, 22, 4, 5.123)],
+      ['2006-01-02T22:04:05.1234Z', DateTime.new(2006, 1, 2, 22, 4, 5.1234)],
+      ['2006-01-02T22:04:05.12345Z', DateTime.new(2006, 1, 2, 22, 4, 5.12345)],
+      ['2006-01-02T22:04:05.123456Z', DateTime.new(2006, 1, 2, 22, 4, 5.123456)],
+      ['2002-10-02T10:00:00-05:00', DateTime.new(2002, 10, 2, 10, 0, 0, Rational(-5, 24))],
```

Comment:
I think using `Rational` doesn't make this test as readable, could we instead just have the expected time in UTC for this and the one below?
```suggestion
      ['2002-10-02T10:00:00-05:00', DateTime.new(2002, 10, 2, 15, 0, 0)],
```

Reply:
This is the same in PHP - we just create a `\DateTime` object with the timezone provided, then the test just formats with that timezone/offset https://github.com/getyoti/yoti-php-sdk/pull/116 - should we change that too?

Or perhaps the Ruby tests could always provide `'+00:00'` for the other test examples?

Reply:
Ah ok, I guess as long as we have testing elsewhere which is checking that providing a timestamp with a (non-UTC) timezone is converted to the correct UTC time, then we don't need to change these ones.